### PR TITLE
Prevent FORMS_URLFIELD_ASSUME_HTTPS warning on Django 5.0

### PR DESCRIPTION
**Pull request by adamchainz**
Prevent this warning:

```
$ python -Wall manage.py help
/.../django/conf/__init__.py:214: RemovedInDjango60Warning: The FORMS_URLFIELD_ASSUME_HTTPS transitional setting is deprecated.
  warnings.warn(
...
```

The approach of listing the setting in `deprecated_settings` is the same as for `STORAGES` in #349, but it does not need to be conditional.

### CODE REVIEW

**Comment by adamchainz** [comment]
Tests are passing but Codecov is failing to upload:

```
[2024-05-09T10:11:52.172Z] ['error'] There was an error running the uploader: Error uploading to https://codecov.io: Error: There was an error fetching the storage URL during POST: 429 - {'detail': ErrorDetail(string='Rate limit reached. Please upload with the Codecov repository upload token to resolve issue. Expected time to availability: 925s.', code='throttled')}
```

I think this needs an admin to reinstall the app or grab a token from codecov.io. Or we could ditch codecov per https://hynek.me/articles/ditch-codecov-python/ - that’s what I did for my open source projects.

**Comment by pauloxnet** [comment]
> Tests are passing but Codecov is failing to upload:
> 
> ```
> [2024-05-09T10:11:52.172Z] ['error'] There was an error running the uploader: Error uploading to https://codecov.io: Error: There was an error fetching the storage URL during POST: 429 - {'detail': ErrorDetail(string='Rate limit reached. Please upload with the Codecov repository upload token to resolve issue. Expected time to availability: 925s.', code='throttled')}
> ```
> 
> I think this needs an admin to reinstall the app or grab a token from codecov.io. Or we could ditch codecov per https://hynek.me/articles/ditch-codecov-python/ - that’s what I did for my open source projects.

Thanks @adamchainz I opened an issue about that https://github.com/jazzband/django-configurations/issues/390

**Review by pauloxnet — APPROVED** [review]
LGTM

**Comment by uhurusurfa** [comment]
@pauloxnet - do you have permissions to rebase this PR so it reruns the tests now that the codecov issue is resolved?

**Comment by uhurusurfa** [comment]
@pauloxnet  - can you also confirm if you have been given full maintainer permissions on this project yet please

**Comment by pauloxnet** [comment]
> @pauloxnet - can you also confirm if you have been given full maintainer permissions on this project yet please

@uhurusurfa please avoid tagging me 2 times in 1 minute in 2 separate comments, as I already wrote I don't have full maintainer permission and I can't access the settings or force a rebase.

In future when you tag me in an issue please wait for a response, I'll try to reply when I'll find the free time to do that, it can be days, I understand your enthusiasm, but this is a total volunteer and unpaid job for me, and I can't assure any time of response or unlimited amount of time in it.

In the alternative, tag other maintainers, or apply to be part of the project.

**Comment by uhurusurfa** [comment]
> > @pauloxnet - can you also confirm if you have been given full maintainer permissions on this project yet please
> 
> @uhurusurfa please avoid tagging me 2 times in 1 minute in 2 separate comments, as I already wrote I don't have full maintainer permission and I can't access the settings or force a rebase.
> 
> In future when you tag me in an issue please wait for a response, I'll try to reply when I'll find the free time to do that, it can be days, I understand your enthusiasm, but this is a total volunteer and unpaid job for me, and I can't assure any time of response or unlimited amount of time in it.
> 
> In the alternative, tag other maintainers, or apply to be part of the project.

Fair enough. I will try alternative roputes

**Review by uhurusurfa — APPROVED** [review]
🚀

**Pull request merged**
